charts/spark-operator-chart/templates/rbac.yaml

```
@@ -13,14 +13,20 @@ rules:
 - apiGroups:
   - ""
   resources:
+  - configmaps
   - pods
   verbs:
   - "*"
+- apiGroups:
+  - coordination.k8s.io"
```

I guess you have a typo here:
```suggestion
  - coordination.k8s.io
```

---

@gagarinfan You guess right 👍